faker/providers/bank/__init__.py

```diff
@@ -1,3 +1,4 @@
+from math import ceil
```

Comment:
we use `isort` to sort imports. Can you move this import down to Line 5? (there should be a blank line between the `import ...` block and the `from ... import ...` block)